Taxi/Taxi/Presenter/TaxiPartyList/TaxiPartyList.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  TaxiPartyList.swift
+//  Taxi
+//
+//  Created by JongHo Park on 2022/08/06.
+//
+
+import SwiftUI
+
+// MARK: - List View
+struct TaxiPartyList: View {
+    @ObservedObject private var viewModel: TaxiPartyList.ViewModel
+    @EnvironmentObject private var userState: UserInfoState
+    @State private var showTaxiPartyInfo: Bool = false
+    @State private var showBlur: Bool = false
+    @State private var showAddTaxiParty: Bool = false
+    @State private var scrollTo: Date?
+    @State private var showCalendarModal: Bool = false
+
+    init(_ viewModel: TaxiPartyList.ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ZStack {
+            VStack(spacing: 0) {
+                header()
+                FilterBar($showCalendarModal) {
+                    viewModel.changeFilter($0)
+                }
+                content()
+            }
+
+            CalendarModal(isShowing: $showCalendarModal, renderedDate: $scrollTo, taxiPartyList: viewModel.taxiPartyForCalendar)
+
+        }
+        .navigationBarTitleDisplayMode(.inline)
+        .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .topLeading)
+        .blur(radius: showBlur ? 10 : 0)
+        .fullScreenCover(isPresented: $showAddTaxiParty) {
+            AddTaxiParty(viewModel: viewModel, user: userState.userInfo)
+        }
+    }
+}
+
+// MARK: - View Header
+private extension TaxiPartyList {
+    func header() -> some View {
+        HStack(alignment: .center) {
+            Text("택시팟")
+                .font(.custom("AppleSDGothicNeo-Bold", size: 25))
+                .padding(.leading, 16)
+            Spacer()
+            Button {
+                showAddTaxiParty = true
+            } label: {
+                Image(systemName: "plus")
+                    .imageScale(.large)
+            }
+            .padding(EdgeInsets(top: 0, leading: 32, bottom: 16, trailing: 19))
+        }
+    }
```

Comment:
현재 패딩값이 이상하게 적용되어 있어서 '택시팟' 글씨와 '+' 버튼의 정렬이 맞지 않습니다.
이로 인해 마이팟, 설정 탭의 타이틀과 택시팟 탭의 타이틀의 높이가 달라서 수정이 필요해 보입니다.

| 현재 | 수정시 |
|:---:|:---:|
|<img width="250" src="https://user-images.githubusercontent.com/75792767/185564011-3a79b2ff-ec43-43f0-8770-cce01c3424f5.png">|<img width="250" src="https://user-images.githubusercontent.com/75792767/185564047-040fd108-a5fd-423f-bc28-25b84b65e95b.png">|

<img width="250" src="https://user-images.githubusercontent.com/75792767/185564697-6d8d7149-dd0c-455b-9bf4-8a07efbf01b5.gif">

```suggestion
    func header() -> some View {
        HStack(alignment: .center) {
            Text("택시팟")
                .font(.custom("AppleSDGothicNeo-Bold", size: 25))
            Spacer()
            Button {
                showAddTaxiParty = true
            } label: {
                Image(systemName: "plus")
                    .imageScale(.large)
            }
        }
        .padding(.horizontal)
    }
```

Reply:
제안 감사합니다 바로 적용했습니ㅏㄷ!!!